source/databases-collections.txt

```diff
@@ -119,31 +119,39 @@ initial collection.
 
 To create a new database:
 
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
    database to exist.
 
 #. Click the :icon-fa5:`plus` icon that appears.
 
-#. In the prompt, enter a name for your new database.
+#. A JavaScript file populates in a new tab with const 
+   variables for the ``database`` and ``collection`` names.
 
-#. Press the enter key.
+#. Update the variable for ``database`` with your new database name.
 
-#. Enter a name for the first collection in your new database.
+#. Update the variable for ``collection`` with your new collection name.
```

Comment:
[optional]

Up to you, but you could consider combining these steps into one:

"Update the ``database`` and ``collection`` variables with the names for your database and collection."